mmv1/third_party/terraform/services/securitycentermanagement/resource_scc_management_organization_event_threat_detection_custom_module_test.go.erb

```diff
@@ -90,12 +93,16 @@ resource "google_scc_management_organization_event_threat_detection_custom_modul
 			  ]}
             EOF
 }
+<% end -%>
 `, context)
 }
 
 func testAccSecurityCenterManagementOrganizationEventThreatDetectionCustomModule_sccOrganizationCustomModuleUpdate(context map[string]interface{}) string {
 	return acctest.Nprintf(`
+<% unless version == 'ga' -%>
 resource "google_scc_management_organization_event_threat_detection_custom_module" "example" {
+	provider = google-beta
```

Comment:
`provider = google-beta` need to be added to the tests in the other three files as well.

Reply:
Addressed.